WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -421,10 +420,6 @@ public static void setLoginWizardStyleActive(boolean loginWizardActive) {
         }
     }
 
-    public static boolean isLoginWizardStyleActivated() {
-        return BuildConfig.LOGIN_WIZARD_STYLE_ACTIVE || getBoolean(UndeletablePrefKey.LOGIN_WIZARD_STYLE_ACTIVE, false);
```

Comment:
Can we remove `UndeletablePrefKey.LOGIN_WIZARD_STYLE_ACTIVE` as well perhaps?

Reply:
Yes, we can.  I removed the preference key in 5e8d7b3d5332b6525bab30146b831a7e13a36b3e.